### ISSUE TITLE
Missing Set Window Position endpoint

### ISSUE THREAD

**Issue by drobota** [mozilla/geckodriver#113]
I got error 

```
WebDriverException: Message: POST /session/b77ab907-13bf-c341-a9cd-a516052eecd4/window/current/position did not match a known command
```

Selenium 2.53.5(python)
Firefox Nightly 50.0a1

**Comment by andreastt** [comment]
Marionette [implements this](https://github.com/mozilla/gecko-dev/blob/master/testing/marionette/driver.js#L1250) but it looks like it’s missing from [the specification](http://w3c.github.io/webdriver/webdriver-spec.html#resizing-and-positioning-windows). I reckon we forgot to add this to the specification, which explains why it’s missing from geckodriver/webdriver-rust.

**Comment by andreastt** [comment]
I’ve submitted https://github.com/w3c/webdriver/pull/307 to add _Get Window Position_ and _Set Window Position_ to the WebDriver specification.

**Comment by andreastt** [comment]
The spec change was accepted and merged earlier today. I’ve also submitted https://github.com/jgraham/webdriver-rust/pull/37 to add these endpoints to the WebDriver Rust library, which is a prerequisite for landing it in geckodriver.

**Comment by mgavilan** [comment]
Hi andreastt,

with Firefox 48 just being released and FirefoxDriver no longer supporting this new browser version it becomes very important ot use to be able to use setPosition() with geckodriver.

Also I noticed new Geckodriver version 0.10.0 will only be compatible with Selenium 3, which is actually in beta stadium Unless there is a stable Selenium version, we will not be able to use Selenium 3, so will there be any possibility to get this new setPosition() API for geckodriver but remain using Selenium 2.53.1 (Java)?

Thanks and best regards
mgavilan

**Comment by andreastt** [comment]
@mgavilan I don’t know about your Selenium questions. It sounds like you need to speak to the Selenium project.

We will land support for _Set Window Position_ in geckodriver quite soon.

**Comment by tobli** [comment]
I believe this is fixed via #254, right? Seems to be working with geckodriver 0.11.1. Thanks!
